example/swagger-files/examples.json

```diff
@@ -10,42 +10,63 @@
     "version": "1.0"
   },
   "paths": {
-    "/results": {
```

Comment:
Can you create a new swagger file for the new example or keep the old routes?

Reply:
done